packages/@orbit/jsonapi/src/jsonapi-source.ts

```diff
@@ -72,24 +76,27 @@ export default class JSONAPISource extends Source implements Pullable, Pushable
   // Pushable interface stubs
   push: (transformOrOperations: TransformOrOperations, options?: object, id?: string) => Promise<Transform[]>;
 
+  // Queryable interface stubs
+  query: (queryOrExpression: QueryOrExpression, options?: object, id?: string) => Promise<DeserializedDocument>;
+
   constructor(settings: JSONAPISourceSettings = {}) {
     assert('JSONAPISource\'s `schema` must be specified in `settings.schema` constructor argument', !!settings.schema);
     assert('JSONAPISource requires Orbit.Promise be defined', Orbit.Promise);
-    assert('JSONAPISource requires Orbit.fetch be defined', Orbit.fetch);
+    assert('JSONAPISource requires Orbit.fetch be defined', Orbit.fetch || fetch);
```

Comment:
This test is no longer needed, since JSONAPISource now defaults to using the global `fetch` if `Orbit.fetch` hasn't been defined.